huggingface/docker/Dockerfile-local

```diff
@@ -0,0 +1,93 @@
+# ONNX Runtime Training Module for PyTorch
```

Comment:
Can we just use the docker files in torch/ort repo?

Reply:
We need some additional packages for huggingface. Otherwise Dockerfile/Dockerfile-10.2 are pretty much the same as docker files in torch/ort repo
The 1 gpu error that I mentioned before doesn't always repro so I'll just use 1 dockerfile here.